### PR TITLE
Fixes logging for Parameter Store and Secrets Manager integrations

### LINKED ISSUE #165: Spring Boot logging and missing traces
[test-spring-logging.zip](https://github.com/awspring/spring-cloud-aws/files/6841792/test-spring-logging.zip)
From https://github.com/spring-projects/spring-boot/issues/27406

I think there is a bug / problem in the way Spring Boot configures logging with Logback.

Using Spring Boot 2.4.8 and Spring Cloud Aws 2.31, I created a minimalist Spring Boot application with a dependency on "spring-cloud-starter-aws-parameter-store-config" using paramstore.

If I do not set any AWS credentials, the AWS SDK cannot connect, so there is an error. But Spring Boot only logs :

```
ERROR | org.springframework.boot.diagnostics.LoggingFailureAnalysisReporter              | report                                   | 

***************************
APPLICATION FAILED TO START
***************************
```

I also added a test class without Spring Boot, using Logback and a similar AWS call, and the logs contain :

```
DEBUG | com.amazonaws.auth.AWSCredentialsProviderChain                                   | getCredentials                           | Unable to load credentials from WebIdentityTokenCredentialsProvider: To use assume role profiles the aws-java-sdk-sts module must be on the class path.
DEBUG | com.amazonaws.auth.AWSCredentialsProviderChain                                   | getCredentials                           | Unable to load credentials from com.amazonaws.auth.profile.ProfileCredentialsProvider@5ce4369b: Unable to load credentials into profile [default]: AWS Access Key ID is not specified.
WARN  | com.amazonaws.internal.InstanceMetadataServiceResourceFetcher                    | handleException                          | Fail to retrieve token 
com.amazonaws.SdkClientException: Failed to connect to service endpoint: 
```

So, as you can see all, those AWS logs are missing when using Spring Boot.
[test-spring-logging.zip](https://github.com/spring-projects/spring-boot/files/6841618/test-spring-logging.zip)

### LINKED ISSUE #194: Logging for Parameter Store
**Type**: Bug / Help needed

**Component**:
"Parameter Store"

**Describe the bug**
I ran the sample for spring-cloud-aws-parameter-store and tried adding debug logging in order to see which parameters should be retrieved from SSM.

The [Documentation](https://docs.awspring.io/spring-cloud-aws/docs/2.3.0/reference/html/index.html#integrating-your-spring-cloud-application-with-the-aws-parameter-store) states that

>In order to find out which properties are retrieved from AWS Parameter Store on application startup, turn on DEBUG logging on io.awspring.cloud.paramstore.AwsParamStorePropertySource class.
>
>`logging.level.io.awspring.cloud.paramstore.AwsParamStorePropertySource=debug`

I added that to `application.properties` but I can see no debug logs. Can you please explain how to enable debug logging in your sample? Without that debugging will be a nightmare.

Thank you.

**Sample**
See https://github.com/awspring/spring-cloud-aws/tree/2.3.x/spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description

Fixes logging for Parameter Store and Secrets Manager integrations when used with `spring.config.import`. Implemented in similar way it's been done to Spring Vault integration https://github.com/spring-cloud/spring-cloud-vault/issues/565

Fixes #194 
Fixes #165

### CODE REVIEW

**Review by MatejNedic — APPROVED** [review]

File: spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # importing single parameter
-spring.config.import=aws-parameterstore:/config/spring/
+#spring.config.import=aws-parameterstore:/config/spring/
+logging.level.io.awspring.cloud=debug
```
Comment (MatejNedic) [inline spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties:3]: We should add the same line to the Secrets Manager sample as well

File: spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/pom.xml [hunk 2]
```diff
@@ -17,6 +17,12 @@
 			<artifactId>spring-boot-starter</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-bootstrap</artifactId>
```
Comment (MatejNedic) [inline spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/pom.xml:22]: Is this dependency really needed for a sample?

**Review by MatejNedic — COMMENTED** [review]
Just few questions about samples

File: spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties (hunk 1 above)
Comment (maciejwalkowiak) [reply spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/src/main/resources/application.properties]: good catch

File: spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/pom.xml (hunk 2 above)
Comment (maciejwalkowiak) [reply spring-cloud-aws-samples/spring-cloud-aws-parameter-store-sample/pom.xml]: not at all!

**Pull request merged**